eng/Versions.props

```diff
@@ -10,9 +10,10 @@
     <VersionPrefix>1.0.2</VersionPrefix>
     <VsixVersionPrefix>16.0.28417</VsixVersionPrefix>
     <PreReleaseVersionLabel>beta4</PreReleaseVersionLabel>
+    <MSTestv2IntellitestFrameworkVersion>net472</MSTestv2IntellitestFrameworkVersion>
     <!-- Dependencies -->
     <MicrosoftVisualStudioCoreUtilityVersion>15.0.26201</MicrosoftVisualStudioCoreUtilityVersion>
-    <MicrosoftInternalIntelliTestVersion>15.0.0</MicrosoftInternalIntelliTestVersion>
+    <MicrosoftInternalIntelliTestVersion>17.3.0-beta.22215.7</MicrosoftInternalIntelliTestVersion>
```

Comment:
I guess we would need to release a version of IntelliTest before we can actually release the new version of test-template so that we don't reference a "beta" version.